packages/devtools_app/lib/src/framework/release_notes/release_notes.dart

```diff
@@ -224,10 +227,9 @@ class ReleaseNotesController {
         // version of DevTools.
         _releaseNotesMarkdown.value = null;
         toggleReleaseNotesVisible(false);
-        logger.log(
+        _log.warning(
           'Warning: could not find release notes for DevTools version '
```

Comment:
can the "Warning:" prefix go away now? I believe `_log.warning` puts "[WARNING]" at the font of the log

Reply:
I had opted not to do that, to keep the number of changes in this PR down.
Would you be alright with me doing that as a follow up?